scheduler/data-flow/build.gradle.kts

```diff
@@ -88,3 +89,9 @@ tasks.withType<Jar> {
 application {
     mainClass.set(dataflowMainClass)
 }
+
+
+downloadLicenses {
+    includeProjectDependencies = true
```

Comment:
:thought_balloon: We may want to use the `aliases` [property](https://github.com/hierynomus/license-gradle-plugin#license-reporting), as we appear to have some naming differences in `3rd-party-summary.txt` introduced in this PR, e.g.
```
  8 Apache 2.0
      3 Apache-2.0
    395 Apache License 2.0
      7 Apache License, Version 2.0
```

Happy to do a follow-up PR to tidy this up, as thus far this PR looks good otherwise

Reply:
yes - lets do a follow up. As stated in the description it was difficult to get all the settings working in Kotlin.